src/views/accounts/AccountDiscover/AccountDiscoverList.vue

```diff
@@ -109,6 +110,3 @@ export default {
   }
 }
 </script>
-<style lang='scss' scoped>
-
-</style>
```

Comment:
There seem to be two areas that could potentially benefit from minor improvements:

1. The `@deleteDialog.visible` should ideally read "if deleteDialog.visible" so there is no unnecessary double negation.
2. As for asset tree rendering style, it's not clear what exactly you refer to ("<asset-tree-table>", "<scss-style>", etc.). Could you clarify this?